docker/docker_client.go

```diff
@@ -27,7 +27,7 @@ const (
 	dockerHostname = "docker.io"
 	dockerRegistry = "registry-1.docker.io"
 
-	systemPerHostCertDirPath = "/etc/docker/certs.d"
+	systemPerHostCertDirPath = "/etc/containers/certs.d"
```

Comment:
This is unfortunately a breaking change. What we can do is actually support the Docker cert dir AND this new directory so we don't break anyone.

Reply:
Yes lets support both.  Search /etc/containers/certs.d first and fall back to /etc/docker/certs.d